packages/indexer-common/src/indexer-management/resolvers/allocations.ts

```diff
@@ -484,7 +441,7 @@ export default {
         amount: amount.toString(),
```

Comment:
We could probably log the `allocationAmount` version of the amount

Reply:
> LGTM:) Just a small question about `ActionType.collect` that was previously considered, I guess we don't need it since the worker will handle it along with unallocate and reallocate?

Yeah, collect is just always performed after a successful unnallocate or reallocate action.